src/main/java/codoc/logic/commands/CommandResult.java

```diff
@@ -47,6 +50,19 @@ public CommandResult(String feedbackToUser) {
         this(feedbackToUser, false, false);
     }
 
+    /**
+     * Instantiates a new Command result.
+     *
+     * @param feedbackToUser the feedback to user
+     * @param justAdded      whether a new person is added
+     */
+    public CommandResult(String feedbackToUser, boolean justAdded) {
```

Comment:
Good because it does not overlap with any existing constructors for CommandResult.
(Other constructors take String, plus two or zero booleans)